### DISCUSSION TITLE
Code getting Copied incorrectly

### DISCUSSION THREAD

**Discussion by rahulgavhar** [community/community#68207] [Mobile]
### Select Topic Area

Bug

### Body

If I copy a python file on mobile app,(from 'browse code' section) each line is starting with 1 space at the beginning and This is creating an indentation error while running it. So how can I copy it properly?

**Comment by alcere** [comment]
@rahulgavhar thanks for reporting! In order to properly create an internal tracking issue, can you share with us the app version and device you are using?

**Reply by rahulgavhar** [reply]
The app is the current app 1.132.0. Device Android version 12 , realme ui version V3.0model 
Device name realme narzo 30

**Reply by alcere** [reply]
Thanks for the info! I will create an internal issue to investigate this problem.
